### DISCUSSION TITLE
kick and ban commands not working

### DISCUSSION THREAD

**Discussion by Zeph108-dev** [Rapptz/discord.py#9097] [Q&A]
when i tried to make kick and ban commands the commands only worked on people without roles but the bots were administrators and the roles were cosmetic so the bot should have banned or kicked them but it gave an error and to test if the bots could ban if they were working or not i kicked and banned the user myself and it worked even though i had the same privilege as the bot.

And i already tried a lot of things to make it work

**Comment by IAmTomahawkx** [comment]
As explained in #9096 , it does not matter what permissions the user has. Discord only cares that they have a role above the roles your bot has. To successfully perform a moderation action on the user, move the users role(s) below the bots role (or vice versa)
